react-frontend/src/components/createRecognitionComponent.js

```diff
@@ -0,0 +1,218 @@
+import React, { useState, useReducer, useEffect } from "react";
+import { Form, Button, Col, Row, Card, InputGroup } from "react-bootstrap";
+import reducer from "components/reducers/createRecognitionReducer";
+import { setDetails } from "components/actions/createRecognitionAction";
+import { string, object } from "yup";
+import PropTypes from "prop-types";
```

Comment:
@jitendrasbunde , @mayuriardad 

[React Development Guidelines](https://github.com/joshsoftware/peerly/wiki/React-Development-Guidelines) not met.